playground/index.html

```diff
@@ -1,58 +1,70 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Testing History HTML5</title>
-    <script src="https://polyfill.io/v3/polyfill.min.js?features=default%2Ces2015"></script>
-
-    <style>
-      .router-link-active {
-        color: darkorange;
-      }
-      .router-link-exact-active {
-        color: red;
-      }
-      .long {
-        background-color: lightgray;
-        height: 3000px;
-      }
-      .fade-enter-active,
-      .fade-leave-active {
-        transition: opacity 0.3s ease;
-      }
-      .fade-enter-from,
-      .fade-leave-to {
-        opacity: 0;
-      }
-      .child-view {
-        position: absolute;
-        transition: all 0.3s cubic-bezier(0.55, 0, 0.1, 1);
-      }
-      .slide-left-enter-active,
-      .slide-left-leave-active,
-      .slide-right-enter-active,
-      .slide-right-leave-active {
-        transition: all 0.3s;
-      }
-      .slide-left-enter-from,
-      .slide-right-leave-to {
-        opacity: 0;
-        transform: translate(30px, 0);
-      }
-      .slide-left-leave-to,
-      .slide-right-enter-from {
-        opacity: 0;
-        transform: translate(-30px, 0);
-      }
-
-      .info pre {
-        display: inline;
-      }
-    </style>
-  </head>
-  <body>
-    <div id="app"></div>
-  </body>
-</html>
+
+<head>
```

Comment:
can you keep the old formatting in all HTML files? It will make it easier to review. I'm using Prettier to format the HTML files. I just pushed an npm script `lint:fix` to do it automatically.

Reply:
ok